third_party/2and3/requests/packages/urllib3/__init__.pyi

```diff
@@ -1,8 +1,8 @@
 from typing import Any
 from . import connectionpool
-from . import filepost
-from . import poolmanager
-from . import response
+from . import filepost as filepost
```

Comment:
This is kind of a corner case. PEP is very brief here saying just "everything that does not use ``as name`` is hidden (except for ``import *``)". But probably this is indeed too verbose, I can special case this in mypy. Also probably some clarification in the PEP (with examples) is needed. Would you like making a PR to the ``python/peps`` repo?